plugins/rikki/heroeslounge/classes/discord/Attendance.php

```diff
@@ -126,6 +126,36 @@ public static function migrateDiscordTagsToIds()
           }
         }
       }
+    }
+
+    public static function getDiscordTag($discordId)
```

Comment:
how do we get the discord tag already?  is this functionality not already present in some form in the codebase?

Reply:
We currently get a user's Discord tag by inefficiently checking what they fill in on registration against every single member in our Discord server. Because we now also store a discord id, which is unique to an account, we're able to directly query Discord for user information.